webapp-django/crashstats/settings/__init__.py

```diff
@@ -1,8 +1,6 @@
 from .base import *
 try:
     from .local import *
-except ImportError, exc:
-    exc.args = tuple(
-        ['%s (did you rename settings/local.py-dist?)' % exc.args[0]]
-    )
-    raise exc
+except ImportError:
+    print ('WARNING - no local.py'
+           ' (deprecated, use environment variables instead)')
```

Comment:
Can you move this to just after `from .local import *`? I.e. it's deprecated to use `local.py`.
Also, instead of print can you change to:
```python
import warnings
warnings.warn(
    "Use environment variables instead or a .env file", 
    DeprecationWarning
)
```